wpengine/atlas-blueprint-portfolio
# MTKA-1907: Fix missing menus when using Twenty Twenty-Three default theme

Adjusts the `acm.json` export file and docs to export the `current_theme`, `stylesheet` and `template`.

Without this, the theme mods for Twenty Twenty-Two do not apply if a new site is using Twenty Twenty-Three, so menus are not assigned to the expected locations.

```json
    "wp-options": {
        "category_base": "category",
        "permalink_structure": "\/posts\/%postname%\/",
        "nav_menu_options": {
            "0": false,
            "auto_add": []
        },
        "theme_mods_twentytwentytwo": {
            "nav_menu_locations": {
                "primary": 2,
                "footer": 3
            },
            "custom_css_post_id": -1
        },
        "current_theme": "Twenty Twenty-Two",
        "stylesheet": "twentytwentytwo",
        "template": "twentytwentytwo"
    }
```

On merging this to main, I recommend testing with a fresh site on Atlas infrastructure to confirm menus are working there again.

### Jira
- [MTKA-1907](https://wpengine.atlassian.net/browse/MTKA-1907)

### To test
- Create a fresh site in Local.
- Install ACM and Faust: `wp plugin install --activate atlas-content-modeler faustwp`.
- Import the blueprint file from this branch: `wp acm blueprint import https://github.com/wpengine/atlas-blueprint-portfolio/raw/nc/fix-menus-twenty-twenty-three/acm-blueprint.zip`
- Confirm the menu locations are set correctly at Appearance → Menus:

<img width="708" alt="Screenshot 2023-01-17 at 19 03 56" src="https://user-images.githubusercontent.com/647669/212977265-b559f8a2-6358-4c8a-8902-3613345d0d11.png">


[MTKA-1907]: https://wpengine.atlassian.net/browse/MTKA-1907?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
